web-app/src/app/app/billing/success/page.tsx

```diff
@@ -0,0 +1,52 @@
+import { CheckCircle } from "lucide-react";
+import Link from "next/link";
+import { getTranslations } from "next-intl/server";
+
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+
+interface BillingSuccessPageProps {
+  searchParams: Promise<{
+    session_id?: string;
+  }>;
+}
+
+export default async function BillingSuccessPage({
+  searchParams,
+}: BillingSuccessPageProps) {
+  const t = await getTranslations("App.Billing.Success");
+  const params = await searchParams;
+
+  console.log(
+    "Successfully purchased credits with session id:",
```

Comment:
outdated log?

Reply:
I wanted to read the output and inform the user how much his balance was modified, but decided not to implement more.
6784f4ae63cd53c98eea8c603db35b3d25752e05